etc/docker/Dockerfile

```
@@ -52,12 +52,12 @@ RUN wget -O /etc/X11/xorg.conf http://xpra.org/xorg.conf && \
 ENV SUMO_HOME /usr/share/sumo
 
 # Install requirements.txt
-COPY requirements.txt /tmp/requirements.txt
+COPY ../../requirements.txt /tmp/requirements.txt
```

```suggestion
COPY ./requirements.txt /tmp/requirements.txt
```

Change the exact location depending on whether `SMARTS/requirements.txt` or `SMARTS/etc/setup/requirements.txt` is used.